src/scripts/components/Dropdown.jsx

```diff
@@ -134,7 +134,7 @@ class Dropdown extends React.Component {
   }
 
   render() {
-    const { avatar, block, className, disabled, disableScroll, hideCaret, label, icon, lockLabel, position, reset, size, title, type, wide, onStart, onComplete, onReverseStart, onReverseComplete, showOverflow, wrapperClassName } = this.props;
+    const { avatar, inputBlur, block, className, disabled, disableScroll, hideCaret, label, icon, lockLabel, position, reset, size, title, type, wide, onStart, onComplete, onReverseStart, onReverseComplete, showOverflow, wrapperClassName } = this.props;
```

Comment:
Instead of `inputBlur`, use a variable similar to the html5 `autoFocus` property. Maybe `autoFocusInput` as the property name. Typically, blur is associated with losing focus and here the input doesn't have focus yet so there is nothing to blur.

Once you change it to `autoFocusInput`, you will need to default the value to `true` so that we don't affect other places in the app that depend on this behavior.

Reply:
I have updated it to a default value of true and changed name of prop. But, we will have to make sure it do not affects the other parts of rhinogram frontend as we now need to pass autoFocusInput props to dropdown wherever we required a focus on input as per the UI of our existing application. @alexgsummer

Reply:
@alexgsummer The default behaviour in the app is that the input field inside the dropdown is in the focus state, as it was earlier. But, in that case when I do not want focus on the inputs inside the dropdown menu (as in the issue with datepickers inside the dropdown menu), I am passing a property named as inputBlur to the dropdown, which blurs  the focus set on the inputs. Hence, the other parts of the app are not affected by this update.

Reply:
@alexgsummer I have renamed the prop and I am passing autoFocusInput to false in the dropdown component whenever I need to prevent focus on the inputs. Also whenever we are not passing this prop, the behaviour of the app is not affected. Please review.